components/buildless-serverless/internal/controller/resources/deployment.go

```diff
@@ -67,6 +67,10 @@ func (d *Deployment) name() string {
 	return d.function.Name
 }
 
+func (d *Deployment) userGroup() *int64 {
```

Comment:
It returns a very specific usergroup (10001). What is that mean? If its a default user, than please name it and avoid magic numbers.
Why not having that reflected in the name of that function

Reply:
Renamed func to `podRunAsUserUID` and added comment to clarify why is it 1000 a magical number